tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -187,50 +187,46 @@ Check Container Create Timestamps
 
 *** Test Cases ***
 Upgrade Present in vic-machine
-    ${status}=  Get State Of Github Issue  7497
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 11-01-Upgrade.robot needs to be updated now that Issue #7497 has been resolved
-    # ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux
-    # Should Contain  ${output}  upgrade
+    ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux
+    Should Contain  ${output}  upgrade
 
 Upgrade VCH with unreasonably short timeout and automatic rollback after failure
-    ${status}=  Get State Of Github Issue  7497
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 11-01-Upgrade.robot needs to be updated now that Issue #7497 has been resolved
-    # Log To Console  \nUpgrading VCH with 1s timeout ...
-    # ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux upgrade --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout 1s
-    # Should Contain  ${output}  Upgrading VCH exceeded time limit
-    # Should Not Contain  ${output}  Completed successfully
-    # ${rc}  ${output}=  Run And Return Rc And Output  govc snapshot.tree -vm=%{VCH-NAME}
-    # Should Not Contain  ${output}  upgrade
-
-    # # confirm that the rollback took effect
-    # Check Original Version
+    Log To Console  \nUpgrading VCH with 1s timeout ...
+    ${rc}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux upgrade --debug 1 --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout 1s
+    Should Contain  ${output}  Upgrading VCH exceeded time limit
+    Should Not Contain  ${output}  Completed successfully
+    # we should have no snapshots
+    ${rc}  ${output}=  Run And Return Rc And Output  govc snapshot.tree -vm=%{VCH-NAME}
+    Should Be Empty  ${output}
+    # the appliance is restarting - attempt to wait until it's ready
+    Sleep  90
```

Comment:
what are you trying to do here?  is this restarting behavior new?

Reply:
wait for the VCH to restart -- in master I attempted to use a keyword for such activity, but it failed...so I have a simple sleep

Reply:
did you try:
`Set Environment Variable  DOCKER_API_VERSION  1.23`
before calling Wait for VCH Initialization?